### PR TITLE
roaming touch up + wild car eui pairs

### PR DESCRIPTION
**Pull request by lytzV**

### CODE REVIEW

File: docs/network-iot/lorawan-roaming.mdx [hunk 1]
```diff
@@ -137,14 +137,11 @@ Roaming Console.
...
 ## Advanced Roaming
 
-### Session Key Filter
+There are additional configurations, automations, and data you can access with OpenLNS. You can
+learn more about these in the [Advanced Configuration](/iot/run-an-lns/advanced-configuration)
+section.
 
-The session key filter gives an opertator the ability to accept and reject specific roaming traffic
-on a device level. Leveraging the session key filter is recommended for all operators. Read more
-about the session key filter in the
```
Comment (jthiller) [inline docs/network-iot/lorawan-roaming.mdx]: I'm not sure we should lose the push to leverage SKF here. I'd like to see all operators using it.

File: docs/network-iot/lorawan-roaming.mdx [hunk 2]
```diff
@@ -137,14 +137,11 @@ Roaming Console.
...
-[LNS advanced configuration](/iot/run-an-lns/advanced-configuration#session-key-filter).
-
-### Proxy Service
+## Roamimg Proxy Service
 
 If Helium's OpenLNS Architecture is a deal breaker for you, then you can leverage
 [this service we prototyped](https://github.com/helium/helium-packet-router-ingest), which should
```
Comment (jthiller) [inline docs/network-iot/lorawan-roaming.mdx]: Should probably rewrite this while we're in here

File: docs/network-iot/lorawan-roaming.mdx (hunk 1 above)
Comment (lytzV) [reply docs/network-iot/lorawan-roaming.mdx]: @jthiller edited in 2nd commit

File: docs/network-iot/lorawan-roaming.mdx (hunk 2 above)
Comment (lytzV) [reply docs/network-iot/lorawan-roaming.mdx]: @jthiller edited in 2nd commit

**Review by jthiller — DISMISSED** [review]

**Comment by jthiller** [comment]
@lytzV mind taking a peek at the conflict here?

**Comment by lytzV** [comment]
> @lytzV mind taking a peek at the conflict here?

@jthiller think it's resolved

File: docs/network-iot/lorawan-roaming.mdx [hunk 3]
```diff
@@ -137,17 +137,16 @@ Roaming Console.
...
-
 The session key filter gives an opertator the ability to accept and reject specific roaming traffic
 on a device level. Leveraging the session key filter is recommended for all operators. Read more
-about the session key filter in the
-[LNS advanced configuration](/iot/run-an-lns/advanced-configuration#session-key-filter).
+about the session key filter and other advanced configurations in the
+[Advanced Configuration](/iot/run-an-lns/advanced-configuration) section.
 
-### Proxy Service
+## Roamimg Proxy Service
```
Comment (jthiller) [inline docs/network-iot/lorawan-roaming.mdx]:
```suggestion
## Roaming Proxy Service
```

File: docs/network-iot/lorawan-roaming.mdx [hunk 4]
```diff
@@ -137,17 +137,16 @@ Roaming Console.
...
 
-If Helium's OpenLNS Architecture is a deal breaker for you, then you can leverage
-[this service we prototyped](https://github.com/helium/helium-packet-router-ingest), which should
-provide you with a standard roaming interface. Note that Helium is not in active maintenance of this
+If Helium's OpenLNS Architecture is incompatible with a roaming partner's set up, then they can
```
Comment (jthiller) [inline docs/network-iot/lorawan-roaming.mdx]:
```suggestion
If Helium's OpenLNS architecture is incompatible with a roaming partner's set up, then they can
```

**Review by jthiller — CHANGES_REQUESTED** [review]
Spotted title typo

**Review by jthiller — APPROVED** [review]

**Pull request merged**